hs_core/views/resource_folder_hierarchy.py

```diff
@@ -101,7 +101,8 @@ def data_store_structure(request):
     )
 
 
-def data_store_folder_zip(request):
+def data_store_folder_zip(request, res_id=None, input_coll_path=None,
+                          output_zip_file_name=None, remove_original_after_zip=None):
     """

```

Comment:
@aphelionz Can you explain a bit on why these additional parameters (res_id, etc) need to be included? Can request object be leveraged to include these values?